.travis.yml

```diff
@@ -4,15 +4,10 @@ services:
   - docker
 matrix:
  include:
- - name: "fedora 28 python 2 test suite"
-   env:
-    - python=2
-    - os=fedora:28
-    - package_manager=dnf
- - name: "fedora 28 python 3 test suite"
+ - name: "fedora 30 python 3 test suite"
    env:
```

Comment:
I think this commit can be dropped as we no longer use Travis.

Reply:
Yeah. Dropped